src/spec/sampleData.test.js

```diff
@@ -0,0 +1,55 @@
+const menuList = require('../../helpers/menuGenerator');
+const sampleData = require('../../data/sampleData');
+
+// test sample data array
+describe('sample data array', () => {
+  it('should be and array', () => {
+    expect(Array.isArray(sampleData)).toBe(true);
+  });
+
+  it('should have a length of 119 items', () => {
+    expect(sampleData.length).toEqual(120);
+  });
+  // test individual object from sample data array
+  describe('sample data item', () => {
+    const item = sampleData[0];
+    let itemKeys = Object.keys(item);
+    let expected = ['id', 'name', 'menu'];
+
+    it('should be an object with the correct keys', () => {
+      expect(itemKeys.length).toEqual(3);
+      expect(itemKeys).toEqual(expect.arrayContaining(expected));
+      expect(['id', 'name', 'dog']).not.toEqual(expect.arrayContaining(expected));
+    });
+
+    it('should contain the correct value types at each property', () => {
+      expect(typeof item.id).toBe('number');
```

Comment:
Jest's `.toMatchObject()` might simplify this code a bit... https://facebook.github.io/jest/docs/en/expect.html#tomatchobjectobject

Reply:
Thanks, will look into refactoring when I get a chance